Sitenotice.php

```diff
@@ -31,7 +31,7 @@ function wfGlobalSiteNotice( &$siteNotice, $skin ) {
 					<div data-nosnippet style="padding-top:0.3em; padding-bottom:0.1em;">
 						<div class="floatleft"><img alt="Miraheze Logo" src="https://upload.wikimedia.org/wikipedia/commons/b/b7/Miraheze-Logo.svg" decoding="async" width="50" height="50"></div>
 						<div style="padding-bottom: 15px; font-size: 13pt; font-weight: bold;">
-							Miraheze will be upgrading to the latest version of MediaWiki (1.39) on Saturday, 21 January 2023 from <b><u>22:00 UTC</u></b> to approximately 01:00 UTC on Sunday, 22 January 2023. During this time, you will not be able to save changes to your wiki. Please make sure to save any edits at least 5 minutes before the upgrade begins.
+							Miraheze is currently upgrading to MediaWiki 1.39. The upgrade is scheduled to end at 01:00 (UTC), 22 January, 2023. Until the upgrade ends, you will be unable to edit and the wiki may go offline at moments.
```

Comment:
The message is wrong too. Should say wikis may display errors or offline for short moments.